examples/discord.js

```diff
@@ -1,21 +1,27 @@
 /*
  * This example is a very simple way how to connect a discord bot with a mineflayer bot.
  * For this example you will need discord.js installed. You can install with: npm install discord.js
- * Note that discord.js v12 or newer is required.
+ * This example uses discord.js v13
  * You need to do this before running this example:
  * - You need to get a discord token
  * - You need to get the id of the channel you want to use
  */
 
 if (process.argv.length < 6 || process.argv.length > 8) {
-  console.log('Usage : node discord.js <discord bot token> <channel id> <host> <port> [<name>] [<password>]')
-  process.exit(1)
+  console.log('Usage : node discord.js <discord bot token> <channel id> <host> <port> [<name>] [<password>]');
+  process.exit(1);
 }
 
-// Load discord
-const Discord = require('discord.js')
-const client = new Discord.Client()
-let channel = process.argv[3]
+// Load discord.js
+const Discord = require('discord.js');
```

Comment:
can you just destructure this for Intents and Client

Reply:
Yep